### PR TITLE
Subtype indication can be expanded name as well

### PR DESCRIPTION
**Pull request by xbauch**
so we extend the case analysis.

### CODE REVIEW

**Comment by chrisr-diffblue** [comment]
This should be able to run CI without it timing out once https://github.com/diffblue/gnat2goto/pull/229 is merged.

File: experiments/golden-results/SPARK-tetris-summary.txt [hunk 1]
```diff
@@ -13,9 +13,9 @@ Calling function: Process_Pragma_Declaration
 Error message: Unsupported pragma: Volatile
 Nkind: N_Pragma
 --
-Occurs: 2 times
+Occurs: 1 times
 Calling function: Process_Declaration
-Error message: Representation clause declaration
+Error message: Representation clause unsupported: component_size
```
Comment (martin-cs) [inline experiments/golden-results/SPARK-tetris-summary.txt]: How is this fix related to the change in this PR?  It seems vaguely like the change in the golden results for the `'Size` patch...
